clio-client/src/main/scala/org/broadinstitute/clio/client/commands/MoveWgsUbamCommand.scala

```diff
@@ -31,63 +28,96 @@ object MoveWgsUbamCommand
     config: BaseArgs,
     ioUtil: IoUtil
   )(implicit ec: ExecutionContext): Future[HttpResponse] = {
-    config.location.foreach {
-      case "GCP" => ()
-      case _ =>
-        throw new Exception("Only GCP unmapped bams are supported at this time")
-    }
-    config.ubamPath.foreach {
-      case loc if loc.startsWith("gs://") => ()
-      case _ =>
-        throw new Exception("The destination of the ubam must be a cloud path")
-    }
     for {
-      wgsUbam <- queryForWgsUbam(webClient, config) withErrorMsg
-        """Could not query the WgsUbam. There may have been the incorrect number of WgsUbams returned.
-          |No files have been moved""".stripMargin
-      _ <- copyGoogleObject(wgsUbam.ubamPath, config.ubamPath, ioUtil) withErrorMsg
-        "An error occurred while moving the files in google cloud. No files have been moved"
+      _ <- verifyPath(webClient, config) withErrorMsg
+        "Clio client can only handle cloud operations right now."
+      wgsUbamPath <- queryForWgsUbamPath(webClient, config) withErrorMsg
+        "Could not query the WgsUbam. No files have been moved."
+      _ <- copyGoogleObject(wgsUbamPath, config.ubamPath, ioUtil) withErrorMsg
+        "An error occurred while copying the files in google cloud. No files have been moved."
       upsertUbam <- upsertUpdatedWgsUbam(webClient, config) withErrorMsg
         """An error occurred while upserting the WgsUbam. The state of clio is now inconsistent.
```

Comment:
This doesn't read quite right to me. Clio's state (which I read is its internal state) is still totally consistent. What's inconsistent is the state of the world outside of clio.

If we wanted to try and fix that here, we could add code that deletes the copied object in GCS. That would leave things in the same state that they started out in, before the command was executed.

Reply:
I was using "clio's state" as the state of not only the application, but also the external state of ubam locations. I agree its a bit ambiguous. I'll change it.